src/zenml/step_operators/step_executor_operator.py

```diff
@@ -128,6 +129,50 @@ def _resolve_user_modules(
 
         return main_module_path, step_source_path
 
+    @staticmethod
+    def _get_step(
+        pipeline_node: pipeline_pb2.PipelineNode,
+    ) -> "BaseStep":
```

Comment:
How is this related to the Step Resource story? Drive-by improvement?

Reply:
I use the step instance to fetch the resource config below. But actually that whole code can go now that the resource configuration is also in the MLMD contexts. I'll remove it :)